src/RequestMatcher.php

```diff
@@ -164,6 +164,24 @@ public static function getSiteVariables($site = NULL) {
     return $vars;
   }
 
+  /**
+   * Gets the variant url.
+   *
+   * @param string $variant
+   *   Variant name.
+   *
+   * @return bool|string
+   *   Variant url or bool if variant incorrect.
+   */
+  public function getHostForSiteVariant($variant) {
+    if (!in_array($variant, $this->variants)) {
+      return FALSE;
```

Comment:
should be "NULL" then, avoid different types for return variables.
or even better would be to throw some exceptoin, like invalidArgumentException